### ISSUE TITLE
Group form fields in columns

### ISSUE THREAD

**Issue by newtec-foggia** [Formfeed-UK/nova-theming-classes#1]
sorry, it is not clear to me, in the resource I have more fields, once the package is installed how do I proceed?

thanks,

**Comment by ianrobertsFF** [comment]
The package only adds css classes to the Nova and 3rd party package vue components.

In order to achieve any specific functionality as a result you need to write css that targets those components. 

For example you could do something like:

```css
.resource-{your resource}.component-form-panel > .component-card {
    display:grid;
    grid-template-columns: 50% 50%;
}
```

You can inspect the html of the page to see specifically what css classes you are working with, but they follow the pattern described in the docs.

**Comment by ianrobertsFF** [comment]
Looks like you already received an answer here: https://github.com/laravel/nova-issues/discussions/4534

What part, specifically, are you struggling with?

**Comment by newtec-foggia** [comment]
I am not very experienced, I have installed the component through composer but I have difficulty and I do not know how to proceed to add the class to the fields in the resource, I am stuck on defining the fields, do you have any example to turn to that allows you to have more fields on the same line?

**Comment by ianrobertsFF** [comment]
You don't need to do anything to add the classes, they're automatic. Once the package is installed the classes will automatically be applied. Just define your fields as you normally do, and when you navigate to your resource in a browser and inspect the html and you'll see that the css classes are applied.

Anything that starts `component-`, `field-`, `resource-` are the css classes applied by this package. 

You then need to write your own css, using those classes, in order to style your forms how you want them to be styled.

Unfortunately it's outside the scope of this package to teach you how to write CSS, the answer you received on the nova-issues discussion I linked gave you the basics, as did the snippet I posted earlier.
